conda-recipe/run_test.bat

```diff
@@ -0,0 +1,25 @@
+ 
+set DAAL4PY_VERSION=%PKG_VERSION%
+set MPIROOT=%PREFIX%\Library
+
+IF DEFINED DPCPPROOT (
```

Comment:
Are we sure that the DPCPP compiler is required as runtime always?

Reply:
it's not required - but in case we have dpcpproot passed - it will be used

Reply:
and if it is not? for example, when testing a cpu part where there is no runtime.